src/Blockcore.Indexer/Storage/Mongo/Types/MapRichlist.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Blockcore.Indexer.Storage.Mongo.Types
+{
+  public class MapRichlist
+   {
+      #region Public Properties
```

Comment:
Never ever use regions. Always delete regions if you find them. Regions are evil: https://marcduerst.com/2016/10/03/c-regions-are-evil/